php/php.editor/test/unit/data/goldenfiles/org/netbeans/modules/php/editor/parser/ASTPHP5ParserTest/parser/heredoc00.pass

```diff
@@ -42,11 +42,11 @@
         <token id='T_VARIABLE' start='69' end='73'>
             <text>$age</text>
         </token>
-        <token id='T_ENCAPSED_AND_WHITESPACE' start='73' end='84'>
-            <text>&lt;br&gt;\nHEREDO</text>
+        <token id='T_ENCAPSED_AND_WHITESPACE' start='73' end='78'>
+            <text>&lt;br&gt;\n</text>
         </token>
-        <token id='T_END_HEREDOC' start='84' end='85'>
-            <text>C</text>
+        <token id='T_END_HEREDOC' start='78' end='85'>
+            <text>HEREDOC</text>
```

Comment:
Token texts were not correct like this.
https://github.com/apache/incubator-netbeans/blob/4fe72ac68f1b22311365b3e91315d4c267c9d638/php/php.editor/test/unit/data/testfiles/parser/heredoc00.php#L1-L24